source/secure_tunneling.c

```diff
@@ -518,6 +519,11 @@ struct aws_secure_tunnel *aws_secure_tunnel_new(
     secure_tunnel->stream_id = INVALID_STREAM_ID;
     secure_tunnel->websocket = NULL;
 
+    struct aws_mutex mutex = AWS_MUTEX_INIT;
+    struct aws_condition_variable condition_variable = AWS_CONDITION_VARIABLE_INIT;
+    secure_tunnel->send_data_mutex = mutex;
```

Comment:
you could assign directly without the temps

Reply:
It will not let me do it, unsure why.

Reply:
In that case I'd just call the init functions on the members